src/Elmish/Boot.purs

```diff
@@ -1,35 +1,54 @@
+module Elmish.Boot
```

Comment:
The diff here is a bit confusing, but the change is that I moved the module comment to `BootRecord`. This way it shows up on re-exports too.